<!DOCTYPE html>
<html lang="en">
  <!-- Head tag -->
  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- Title -->
  
  <title>1. Metagenomics - Lily - soil microbiomes blog</title>

  <!--Favicon-->
  <link rel="icon" href="favicon/favicon.ico">

  <!--Description-->
  
      <meta name="description" content="1.Describe the role of ‘omics in microbiome researchMetagenomics is the study of genetic material recovered directly from environmental samples (Wikip">
  

  <!--Author-->
  
      <meta name="author" content="John Doe">
  

  <!-- Pure CSS -->
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
  <link href="https://fonts.googleapis.com/css?family=Crimson+Text|Open+Sans:300,800" rel="stylesheet">

  <!-- Custom CSS -->
  <link rel="stylesheet" href="/css/styles.css">

  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
  <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
  <![endif]-->

  <!-- Google Analytics -->
  

</head>


  <body>
  	<div class="container-fluid navbar-container m-sm-5">
      <!-- Header -->
      <nav class="navbar navbar-toggleable-sm navbar-light px-1 py-3 my-3 mb-sm-5">
  <a class="navbar-brand ml-2" href="/">Lily - soil microbiomes blog</a>
  <button class="navbar-toggler navbar-toggler-right py-2" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>
  <div class="collapse navbar-collapse text-center" id="navbarCollapse">
    <ul class="navbar-nav ml-auto my-auto">
      
        <li class="nav-item">
          <a class="nav-link" href="/about">About</a>
        </li>
      
    </ul>
    <hr class="hidden-md-up" />
  </div>
</nav>


  		<div class="row">
  			<div class="col-12 mb-4">
  <img class="img-fluid project-img" src="/images/image1.jpg" alt="1. Metagenomics">
</div>
<div class="col-lg-4 col-12 pt-3 px-4 pr-lg-5">
  <h1>1. Metagenomics</h1>
</div>
<div class="col-lg-8 col-12 pt-lg-3 mb-4 pl-lg-5 px-lg-0 px-4 portfolio-content">
  <h1 id="1-Describe-the-role-of-‘omics-in-microbiome-research"><a href="#1-Describe-the-role-of-‘omics-in-microbiome-research" class="headerlink" title="1.Describe the role of ‘omics in microbiome research"></a>1.Describe the role of ‘omics in microbiome research</h1><p>Metagenomics is the study of genetic material recovered directly from environmental samples (Wikipedia), and it is one of the most popular ways today to study soil microbes. Metagenomics allows a large number of gene sequences to be recovered in a short period of time. The 16s rRNA information enables us to identify and classify the microbes in the sample, especially those non-culturable ones which do not grow on agar plate and thus are difficult to be found other ways. Full genome metagenomics also enables the identification of functional groups and thus we can predict the possible functionality and distribution of the soil microbes.</p>
<p>Firstly, metagenomics can be used in experiments to simultaneously recover large numbers of gene sequences. Fierer et al. (2012) aimed to test the hypothesis that desert soil microbiomes are taxonomically and functionally distinct compared with those found in other biomes and that the variability across different desert sites is less than the variability between desert and nondesert biomes. Therefore he needed to identify a large number of microbes from multiple samples in order to do the comparison. By using the shotgun metagenomic approach, Fierer and his colleagues were able to obtain a total of 3.9–11 million 100-bp sequences per sample from 16 samples (3 from hot deserts, 6 from Antarctic cold deserts, and 7 from temperate and tropical forests, a prairie grassland, a tundra, and a boreal forest).</p>
<p>Secondly, metagenomics enables the taxonomic identification of soil microbes, especially bacterial species by the 16s rRNA gene. After using shotgun metagenomic approach, Fierer et al. (2012) found that the majority of shotgun metagenomic reads were derived from bacteria. He added on an amplicon sequencing survey of a portion of the 16S rRNA gene  to provide a higher resolution and more indepth analysis of the composition and diversity of the soil bacterial communities. They managed to identify major bacterial groups such as Acidobacteria, Actinobacteria, Bacteroidetes, Proteobacteria, and Verrucomicrobia in the soil by the 16s RNA analysis. With the knowledge of bacterial identity, they were then able to compare the desert microbes with the nondesert microbes. They concluded that microbe composition and diversity are more similar within different dessert environments than outside dessert environments.<br><img src="/images/fig2.png" width="1000px"></p>
<p>Thirdly, metagenomics can be used to identify phenotypic functional genes and thus predict functionality the microbes may have. Fierer et al. (2012) later assessed phenotypic functional genes by metagenomics. Out of the 28 major functional gene categories, 13 of them were significantly different in abundance between desert and nondesert soils. They were able to explain some of the differences such as moisture constraints might have caused nutrient cycling rate to be lower in dessert; and more moisture stress might be the reason why desert soils have higher relative abundances of genes associated with dormancy/sporulation and stress protein.</p>
<img src="/images/fig3.png" width="500px">

<p>Despite all the advantages of metagenomics, there are some limitations associated with it. For example, in Fierer’s experiment (2012), only 13–23% of the obtained sequences using the shortgun metagenomics could be annotated due to reasons such as reads and sequence length limitations. And metagenomics generally have a relative high cost</p>
<p>Overall, despite some subtle drawbacks, metagenomics serves as one the most popular and useful methods in today’s soil microbe studies. It greatly enhances our ability to sequence microbial genes, and enables microbial identity and functionality analysis.</p>

</div>


      </div>
      
  	</div>

    <!-- After footer scripts -->
    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>

  </body>
</html>
